Real-Time Corruptor/BizHawk_RTC/BizHawk.Client.EmuHawk/Program.cs

```diff
@@ -256,10 +256,6 @@ private static int SubMain(string[] args)
 			}
 			catch (Exception e) when (!Debugger.IsAttached)
 			{
-				//RTC_Hijack - ignore AbortEverythingException
-				if (e is RTCV.NetCore.AbortEverythingException)
-					return 0;
```

Comment:
I completely forgot about this. AbortEverythingException() actually is used to ensure any RTC exceptions don't unfurl all the way to Bizhawk's exception handler. This isn't the best way to do this, but we can't kill this code for now.

Reply:
Got it! I'm fine closing this PR for now and leaving the AbortEverythingException in RTCV.